packages/app/obojobo-document-engine/src/scripts/viewer/components/viewer-app.js

```diff
@@ -396,7 +399,7 @@ export default class ViewerApp extends React.Component {
 		}
 	}
 
-	onScroll() {
+	startObservingForIntersectionChanges() {
 		const focusState = this.state.focusState
 
```

Comment:
I think here we should go ahead and kill the existing observer. My thought is you could just call `this.stopObservingForIntersectionChanges()` which would be something like

```javascript
if(!this.observer) return;

this.observer.disconnect();
delete this.observer
```

Reply:
Now that you have the `stopObserving...` method it needs to be called here